client/pages/profile.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import AppContext from '../lib/app-context';
+import Redirect from '../components/redirect';
+
+export default class Profile extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: null,
+      created: 0,
+      completed: 0,
+      style: {
+        width: '10%'
+      }
```

Comment:
Only put values that change in state. You can make this a variable or use CSS like normal.